site/content/3.12/about-arangodb/features/community-edition.md

```diff
@@ -142,12 +142,18 @@ see [arangodb.com/community-server/](https://www.arangodb.com/community-server/)
   for geo-spatial queries.
 
 {{% comment %}} Experimental feature
-- [**Query result spillover**](../../aql/how-to-invoke-aql/with-arangosh.md#spilloverthresholdmemoryusage)
+- [**Query result spillover**](../../aql/how-to-invoke-aql/with-arangosh.md#spilloverthresholdmemoryusage):
   AQL queries can store intermediate and final results temporarily on disk
   (also known as external result sets) to decrease memory usage when a specified
   threshold is reached.
 {{% /comment %}}
 
+{{% comment %}} Experimental feature
+- [**Vector search**](#TODO):
```

Comment:
question: Is the `#TODO` intentional here?

Reply:
Yes, the bullet point is commented out (only GA features are listed) but I wanted to add the entry already as a reminder. There is no page to link to yet.